stats/stats.go

```diff
@@ -37,6 +37,9 @@ type RPCStats interface {
 }
 
 // Begin contains stats when an RPC attempt begins.
+// This event is called AFTER the InHeader event, as headers must
```

Comment:
why is AFTER caps?

Also, you don't have to begin from next line unless you reaching the max columns 80 or if you want to have a new line then you have to put a blank line in between two sentences.

Reply:
nit: `InHeader`

Reply:
Got it! Cleaning this up in the next commit.